hqrentalsoftware/wordpress-hq
# [#30] - Show price with taxes

### Ticket
#30 

### Description
We implemented a way to display the prices of the vehicles on the website with and without taxes depending on the following setting from the system:

URL: .../settings/car-rental/charges-and-fees
![image](https://user-images.githubusercontent.com/51130063/214840074-524fc9f6-f763-4db2-830d-f649059a6b8a.png)


### What does this PR change?
The following changes were made:
- [x] We add the new setting (```car_rental_show_prices_including_sales_tax```) to the setting table.
- [x] We added a new option for the metric system. This is used to parse the price with the correct decimals and thousands separator
- [x] We added the abb tax to the Brands model, a new field is created on the Brand table to keep track of this
- [x]  Recalculate the rates in case it's needed on Query Class, this way it will be propagated on whatever interaction with the vehicle class models.
- [x] A couple of files were deleted, since they are unused.

### How to QA this PR?
On a Wordpress instance properly set, If the option

![image](https://user-images.githubusercontent.com/51130063/214841525-8af45ad1-43f7-498d-b144-69166eb4d53d.png)

You should see the prices without tax

![image](https://user-images.githubusercontent.com/51130063/214841632-507f37a9-2cc4-4f74-8149-13803e79be87.png)


In the other case:

![image](https://user-images.githubusercontent.com/51130063/214841717-590daf6d-3bf7-4242-a84d-8536d7ab830a.png)

The tax will be added to the prices

![image](https://user-images.githubusercontent.com/51130063/214841893-272ef809-57b6-48b0-a415-f03a863929ad.png)

PS: If changes are made in the setting from the system, click on the save button on the settings page from Wordpress, this way the synchronization routine will be fire and you're ready to the test with the new changes.

URL: website.com/wp-admin/admin.php?page=hq-wordpress-settings

![image](https://user-images.githubusercontent.com/51130063/214842382-28c24326-6e2f-42e9-b642-944e519d70f3.png)
